### PR TITLE
UIIN-2124: The "Instance" record still displayed at the third pane when user selects "Browse contributors" option

### PR DESCRIPTION
**Pull request by ODanyor**
## Purpose
The "Instance" record still displayed at the third pane when user selects "Browse contributors" option

## Approach
Redirect user to `inventory/` on `qindex` change to any browse options to close record view window.

## Issues
[UIIN-2124](https://issues.folio.org/browse/UIIN-2124)

### CODE REVIEW

File: src/components/InstancesList/InstancesList.js [hunk 1]
```diff
@@ -967,6 +968,7 @@ class InstancesList extends React.Component {
       if (isBrowseOption) {
         parentMutator.browseModeRecords.reset();
         this.setState({ isSingleResult: false });
+        goTo(path);
```
Comment (BogdanDenis) [inline src/components/InstancesList/InstancesList.js:971]: Will this also close the record when switching from one browse option to another browse option?

Comment (ODanyor) [reply src/components/InstancesList/InstancesList.js:971]: I was worrying  about that too. But the list items don't open any record, they redirect user to non browse options. Correct me If I am wrong

**Review by BogdanDenis — APPROVED** [review]

File: src/components/InstancesList/InstancesList.js (hunk 1 above)
Comment (Dmytro-Melnyshyn) [inline src/components/InstancesList/InstancesList.js:971]: I think it is necessary to leave the `qindex` param in the URL.

Comment (ODanyor) [reply src/components/InstancesList/InstancesList.js:971]: Yes, you are right. Thank you for noticing that!

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**